unblob/finder.py

```diff
@@ -52,6 +52,8 @@ def search_chunks_by_priority(  # noqa: C901
             for offset, identifier, string_data in sorted_match_strings:
                 real_offset = offset + handler.YARA_MATCH_OFFSET
 
+                if real_offset < 0:
+                    continue
```

Comment:
Just a quick question:

The empty `continue` is enough here?
I mean, this is not a "real error" which should be logged and change the exit code of the whole `unblob` from something other than `0`, but just a "precheck" that the given `handler` should not deal with this `offset`, right?

Reply:
Yes this is a pre-check in order to discard it if it's not valid. I put an inline comment on top of it to make things clear.

Reply:
Thank you, just wanted to make sure.